sdk/apis/kubermatic/v1/cluster.go

```diff
@@ -267,6 +268,9 @@ type ClusterSpec struct {
 	// Kyverno holds the configuration for the Kyverno policy management component.
 	// Only available in Enterprise Edition.
 	Kyverno *KyvernoSettings `json:"kyverno,omitempty"`
+
+	// Optional: AuthorizationConfig to configure the apiserver authorization modes
```

Comment:
This PR looks great, sorry for the slow review on it. Given that we don't have tests for this, can you please add a note here (and regenerate the CRD) that this is an experimental feature for now?

```suggestion
	// Optional: AuthorizationConfig to configure the apiserver authorization modes. This feature is experimental right now
```

Reply:
No Problem :) Added and executed codegen